kamernet.js

```diff
@@ -85,27 +120,22 @@ async function urlsOfUnwatchedAdverts(page, pageNum) {
     })
 }
 
-
 export async function lookForRooms() {
+    let browser, page;
     try {
-        const browser = await puppeteer.launch({
-            headless: true,
-            // args: [
-            //     '--no-sandbox',
-            //     '--disable-setuid-sandbox'
-            // ]
+        browser = await puppeteer.launch({
+            headless: false, // set it false to see chrome
```

Comment:
not sure if this makes it faster, but it sure does make the process more friendly/visual

I set it to true cause there is a chance someone could click on the tab and ruin the algorithm's run

Reply:
Okay, just make sure to set the width of the viewport then, since in headless mode it cannot start in maximized view and the login button wouldn't be displayed.